### PR TITLE
Mohamed and Praises

### PR DESCRIPTION
**Pull request by CakeInTech**
## Popup Cards

- You should implement the following interactions:
   - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
   - You should implement popups for both mobile and desktop screen sizes

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    demoLink: '',
+  },
+];
+
+function showModal(work) {
+  const projectDetails = popup[work];
+  document.querySelector('.tonic').innerHTML = projectDetails.title;
+  document.querySelector('.descriptionist').innerHTML = projectDetails.discription;
+  document.querySelector('.work-banners-model').src = projectDetails.imageLink;
+  // document.querySelector('.model .button-list').innerHTML = listButton (popup, work);
```
Comment (Nemwel-Boniface) [inline popup.js]: - [ ] Kindly note that according to the javascript [best practices for the browser](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) number 7, You are supposed to remove all commented and unwanted code from your code base as it makes it cluttered and messy. Kindly make sure that you remove them from your code base.

File: popup.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
+.model {
```
Comment (Nemwel-Boniface) [inline popup.css:1]: - [ ] Great job styling your modal popup it looks really good. However, kindly note that you are supposed to stick as much as possible to the chosen Figma design. Currently, your modal popup for desktop is fixed at the bottom side of the screen and it is supposed to be placed on the middle side of the screen. Kindly fix this.

**This is how your design currently loos like:**

image

**This is the desired Figma design:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @MohamedCK           :wave: , 

I'm Nemwel your reviewer for this PR :man_beard: 

You  have done a great job setting your portfolio modal popup it looks very professional :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: popup.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  filter: blur(10px);
+  z-index: 1;
+}
+
+@media (min-width: 768px) {
+  .model {
+    margin: 0 auto;
+    width: 45.75rem;
+
+    /* height: 50%; */
```
Comment (1GORDON) [inline popup.css]: Great job team 🤝

- [x] [Optional] You may kindly remove all commented code from your style sheets. This is not regarded as a good practice.

File: popup.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    gap: 20px;
+  }
+
+  .lister {
+    font-size: 12px;
+  }
+
+  .model-container {
+    max-width: 60.75rem;
+  }
```
Comment (1GORDON) [inline popup.css:118]: Great job @MohamedCK & @PraisesPJMT 🤝

- [x] Just like the previous reviewer pointed out, you may kindly fix the popup on both mobile and desktop designs. Your popups are both fixed to the bottom. You cold add `margin-botton` to your popups.
- Your mobile popup
mobile popup

- Your desktop popup
desktop popup

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @MohamedCK & @PraisesPJMT ✋

Good job so far!

## Highlights!
- Followed Git flow ✔️ 
- Good commit messages ✔️
- Descriptive Pull Request ✔️
- Linters are passing ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [x] [Optional] You may kindly remove the `locals.js` file from your project as the file is blank and not necessary for now.
- [x] [Optional] You may also use the correct [Readme file template](https://github.com/microverseinc/readme-template) and customize it accordingly. the template you used is different from the one provided in the general requirements.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._** 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi Team, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title :heavy_check_mark:
 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**